editor/src/clj/editor/gui.clj

```diff
@@ -2191,9 +2191,7 @@
   (property def g/Any (dynamic visible (g/constantly false)))
   (property background-color types/Color (dynamic visible (g/constantly false)) (default [1 1 1 1]))
   (property visible-layout g/Str (default (g/constantly ""))
-            (dynamic visible (g/constantly false))
-            (dynamic edit-type (g/fnk [layout-msgs] {:type :choicebox
-                                                     :options (into {"" "Default"} (map (fn [l] [(:name l) (:name l)]) layout-msgs))})))
+            (dynamic visible (g/constantly false)))
```

Comment:
It's concerning that `edit-type` dynamic of `visible-layout` makes it a successor of `visible-layout` output, but most `edit-type`s are harmless `(s/constantly ...)`, so I guess it's fine.